core/src/main/scala/akka/persistence/cassandra/EventsByTagSettings.scala

```diff
@@ -27,6 +27,12 @@ import com.typesafe.config.Config
   case object Max extends Period
   case class Fixed(duration: FiniteDuration) extends Period
 
+  final case class RetrySettings(
+      retries: Int,
+      minDuration: FiniteDuration,
+      maxDuration: FiniteDuration,
+      randomFactor: Double)
```

Comment:
Latest Akka includes `akka.stream.RestartSettings`. Maybe we can use it here?

Reply:
Okay. We don't have a reset as it is for a single query but we can still use it and ignore that

Reply:
Actually, there's no ctr that works for us